superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -220,6 +238,15 @@ int ParseOpts(int argc, char **argv, Opts *opts) {
         case static_cast<int>(OptIdx::kEnableDToD):
             opts->dtod_enabled = true;
             break;
+        case static_cast<int>(OptIdx::kEnableOneToAll):
+            opts->one_to_all_enabled = true;
+            break;
+        case static_cast<int>(OptIdx::kEnableAllToOne):
+            opts->all_to_one_enabled = true;
+            break;
+        case static_cast<int>(OptIdx::kEnableAllToAll):
```

Comment:
What happens if [--dma_copy] + [--all_to_all]

Reply:
--dma_copy and --sm_copy is only effective for dtoh/htod/dtod. For new all_to_all and other options it's always SM.